dashboard/src/containers/AppViewContainer/AppViewContainer.tsx

```diff
@@ -31,6 +31,8 @@ function mapDispatchToProps(dispatch: ThunkDispatch<IStoreState, null, Action>)
     deleteApp: (releaseName: string, ns: string, purge: boolean) =>
       dispatch(actions.apps.deleteApp(releaseName, ns, purge)),
     getApp: (releaseName: string, ns: string) => dispatch(actions.apps.getApp(releaseName, ns)),
+    receiveResource: (payload: { key: string; resource: IResource }) =>
```

Comment:
This is temporary while we move the websockets handling outside of this component/container right? If that's the case I'd add a `TODO` note in this code and also inside the component.

Reply:
Yep, good shout - I'll add a TODO!